### PR TITLE
Transfer Employees to Users [GG-62]

### PR DESCRIPTION
**Pull request by kemptonk**
Converted the Employee model into a custom User class. Updated the tests and API to work with this new model.

### CODE REVIEW

**Comment by lb-cake** [comment]
Should we also change change the API endpoints from employee to user?

Screen Shot 2019-07-20 at 9 41 38 PM

**Comment by kemptonk** [comment]
I don't think we need to change the name of the endpoints since we didn't change the name of the actual model.

File: GGProject/tests/test_api_emp.py [hunk 1]
```diff
@@ -5,149 +5,104 @@
...
     # Delete single employee
     def delete_employee(self, emp_ID):
 
         url = str(self.live_server_url) + "/api/"
 
-        try:
-            response = req.delete(url + "employees/" + str(emp_ID) + "/")
-            response.raise_for_status()
-        except req.exceptions.HTTPError as err:
-            print(err)
```
Comment (mraynor) [inline GGProject/tests/test_api_emp.py:117]: We may not want to print the error any more but do we still want to wrap this in a `try` block?

Comment (kemptonk) [reply GGProject/tests/test_api_emp.py:117]: Wouldn't that prevent the test from returning an error if something went wrong?

Comment (mraynor) [reply GGProject/tests/test_api_emp.py:117]: If you wanted a more specific error message you could use `raise` and use a specific, possibly more useful error message, but if you want the error to be returned then I guess not having the `try` block is right

Comment (kemptonk) [reply GGProject/tests/test_api_emp.py:117]: Luckily the raise_for_status function will display the specific error by default.

**Review by lb-cake — APPROVED** [review]

**Pull request merged**